website/src/middlewares/api-v3/analytics.js

```diff
@@ -0,0 +1,27 @@
+import nconf from 'nconf';
+import {
+ track,
+ trackPurchase,
+ mockAnalyticsService,
+} from '../../libs/api-v3/analyticsService';
+
+function _generateService () {
+  let service;
+
+  if (nconf.get('IS_PROD')) {
+    service = {
+      track,
+      trackPurchase,
+    };
+  } else {
+    service = mockAnalyticsService;
+  }
+
+  return service;
+}
+
+export default function attachAnalytics (req, res, next) {
+  res.analytics = _generateService();
```

Comment:
I was thinking that too. This was mainly for being able to test prod and non-prod. We could memo-ize it.